### PR TITLE
Repo config, yarn, circleci, shipit

### PR DESCRIPTION
**Pull request by GoodForOneFare**
Small repo, so I've just bundled everything together.

From a "how would this work in shipit?" perspective, `scripts/deploy.js` having to be es5 is interesting.  Would a formal shipit integration require everything executed by shipit to be es5?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.DS_STORE
+npm-debug.log*
```
Comment (lemonmade) [inline .gitignore]: `*.log`

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1 @@
+npm-debug.log*
```
Comment (lemonmade) [inline .npmignore]: Same as above

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      fi
+  override:
+    - yarn install
+  cache_directories:
+    - ~/.yarn
+    - ~/.cache/yarn
+
+test:
+  override:
+    - yarn check
```
Comment (lemonmade) [inline circle.yml]: `yarn run ...`

File: scripts/deploy.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+var exec = require('child_process').exec;
```
Comment (lemonmade) [inline scripts/deploy.js]: Simplify deploy script

**Review by lemonmade — CHANGES_REQUESTED** [review]
Same comments as other repos

**Comment by GoodForOneFare** [comment]
All comments addressed.  Should be good to go.

**Review by lemonmade — APPROVED** [review]

**Pull request merged**